doc/dev/release.md

```diff
@@ -306,11 +307,24 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**Note:**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
 You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
 
-### (Post-release) Updating the operator-sdk-samples repo
+### 6. Update the Upgrade version guide document
+
+1. The [Upgrade version guide][version-upgrade-guide] should be updated with the steps required for the users are able to migrate their project.
```

Comment:
```suggestion
1. A new section for upgrading operator projects to v1.3.0 should be added to the [version upgrade guide][version-upgrade-guide] _only if_ there are breaking scaffold or API changes between v1.2.0 and v1.3.0. This section should contain steps directing users to upgrade dependency versions in `go.mod`, update function signatures, etc.
```

Reply:
This section should be [discussed here](https://github.com/operator-framework/operator-sdk/blob/3b2f684/doc/dev/release.md#release-notes).

Reply:
Hi @estroz, 

Regards:
> 1. A new section for upgrading operator projects to v1.3.0 should be added to the [version upgrade guide][version-upgrade-guide] _only if_ there are breaking scaffold or API changes between v1.2.0 and v1.3.0. This section should contain steps directing users to upgrade dependency versions in `go.mod`, update function signatures, etc.

The user always will need to perform changes in order to get the upper version. (at least the go.mod and docker file will be needed to updated) Then, it means that always will be required migration steps for a new version. So, added all less the _only_ statement.